csharp_dotnetcore/13.Basic-Bot-Template/wwwroot/default.htm

```diff
@@ -3,9 +3,10 @@
 <head>
```

Comment:
Edwin submitting a PR last night to update all default.htm pages. can you please pull his change in? It's merged into V4.